pkgs/top-level/all-packages.nix

```diff
@@ -15052,7 +15052,6 @@ in
   indicator-application-gtk3 = callPackage ../development/libraries/indicator-application/gtk3.nix { };
 
   indilib = callPackage ../development/libraries/science/astronomy/indilib { };
-  indi-3rdparty = callPackage ../development/libraries/science/astronomy/indilib/indi-3rdparty.nix { };
```

Comment:
Do we need an alias for this?

Reply:
I don't think so. indi-3rdparty is only useful in the context of being wrapped into an environment with indilib, a use-case covered by indi-full. Same goes for indi-firmware.